addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -148,15 +149,8 @@ if (core.debug) {
             });
         },
         translate: function() {
-            this.do_action({
-                name: _t("Technical Translation"),
-                res_model : 'ir.translation',
-                domain : [['type', '!=', 'object'], '|', ['name', '=', this.dataset.model], ['name', 'ilike', this.dataset.model + ',']],
-                views: [[false, 'list'], [false, 'form']],
-                type : 'ir.actions.act_window',
-                view_type : "list",
-                view_mode : "list"
-            });
+            new Model("ir.translation").call('get_technical_translations', [this.dataset.model]
+                ).then(this.do_action);
```

Comment:
May I suggest better code indentation to suggest the sequence of actions?

    new Model("ir.translation")
        .call('get_technical_translations', [this.dataset.model])
        .then(this.do_action);